clar2wasm/src/standard/standard.wat

```diff
@@ -788,6 +789,158 @@
         )
     )
 
+    (func $lt-buff (type 9) (param $offset_a i32) (param $length_a i32) (param $offset_b i32) (param $length_b i32) (result i32)
```

Comment:
I think it would be really helpful to write out this algorithm in pseudo-code in a comment above this function.

Reply:
Added in 3729fd1